src/Symfony/Component/Messenger/Transport/Doctrine/Connection.php

```diff
@@ -52,12 +52,14 @@ class Connection
     private $configuration = [];
     private $driverConnection;
     private $schemaSynchronizer;
+    private $autoSetup;
```

Comment:
What about calling it `needsSetup` - I think that implies its meaning a bit better?

Reply:
copying redis transport naming here.
also i like that `auto_setup` config option -> `autoSetup` property name